xml/virt_support.xml

```diff
@@ -4,779 +4,274 @@
+
+  <itemizedlist>
+   <title>The level of support is as follows:</title>
    <listitem>
     <para>
-     &sls; 15 (&kvm;/&xen;)
+     Support for SUSE host operating systems is full L3, both for the guest and
+     host.
     </para>
    </listitem>
    <listitem>
     <para>
-     &sls; 15 SP1 (&kvm;/&xen;, when released)
+     Support for 3rd party host environments is full L3 for the guest and needs
+     the host vendor cooperation and support for the host.
     </para>
    </listitem>
   </itemizedlist>
 
+  <note>
+   <title>Support for Third Party Virtualization Hosts</title>
+   <para>
+    &suse; only provides full L3 support if both the virtualization host and
+    guest are using &suse; products.
+   </para>
+   <para>
+    If the virtualization host uses third party software, &suse; only
+    guarantees L3 support for the guest. For the host, the cooperation of the
+    vendor is required.
+   </para>
+  </note>
+ </sect1>
+ <sect1 xml:id="virt-support-guests">
+  <title>Guest Environments</title>
+
   <para>
-   The following third party hypervisors are supported:
+   This section lists the support status for various guest operating systems
+   virtualized on top of &productname; &productnumber;. All guest operating
+   systems are supported both fully-virtualized (<quote>FV</quote> in the
+   following table) and paravirtualized (<quote>PV</quote> in the following
+   table) with two exceptions: Windows, which is only supported
+   fully-virtualized, and NetWare operating systems, which are only supported
+   on &xen; paravirtualized. All guest operating systems are supported both in
+   32-bit and 64-bit flavors, unless stated otherwise (see NetWare).
   </para>
 
   <itemizedlist>
+   <title>The following guest operating systems are fully supported
+    (L3):</title>
    <listitem>
-    <para>KVM for &zseries; 1.1.0</para>
+    <para>
+     SLES 10 SP4
+    </para>
    </listitem>
-   <!-- bsc#984119 -->
    <listitem>
-    <para>IBM PowerVM</para>
+    <para>
+     SLES 11 SP3, SP4
+    </para>
    </listitem>
    <listitem>
-    <!-- according to  http://fate.suse.com/320424 -->
     <para>
-     VMware ESXi 6.0
+     SLES 12 SP0, SP1, SP2, SP3, SP4, SP5
     </para>
    </listitem>
    <listitem>
     <para>
-     VMware ESXi 6.5
+     SLES 15 SP0, SP1, SP2
     </para>
    </listitem>
    <listitem>
     <para>
-     Windows 2008 R2 SP1+
+     OES 11 SP2, 2015, 2015 SP1, 2018, 2018 SP1, 2018 SP2
     </para>
    </listitem>
    <listitem>
     <para>
-     Windows 2012+
+     Netware 6.5 SP8 (32-bit only)
     </para>
    </listitem>
    <listitem>
     <para>
-     Windows 2012 R2+
+     Windows Server 2008 SP2+, 2008 R2 SP1+, 2012+, 2012 R2+, 2016, 2019
     </para>
    </listitem>
+  </itemizedlist>
+
+  <itemizedlist>
+   <title>The following guest operating systems are supported as a technology
+    preview (L2, fixes if reasonable):</title>
    <listitem>
     <para>
-     Microsoft Windows 2016
+     SLED 15 SP1
```

Comment:
Is this the only SLED variant that is tech preview? No SLED 15 SP2 or SLED 12 SPX?

Reply:
Honestly, i took what's here https://confluence.suse.com/display/Virtualization/Support+Constraints without much intervention....if you're sure something's missing there, tell me